interface/src/app/app.js

```diff
@@ -19,15 +20,37 @@ angular.module( 'adage', [
   $resourceProvider.defaults.stripTrailingSlashes = false;
 }])
 
-.run( function run () {
+.run( function run ( $state, $location ) {
+
+    // Will only redirect if there is no state AND the url is not a
+    // route for tribe_client
+    String.prototype.startsWith = function(prefix) {
+        return this.slice(0, prefix.length) == prefix;
+    };
+
+    if ($state['current']['name'] === '') {
+      if (window.location.pathname.startsWith('/tribe_client')) {
```

Comment:
Good question - I actually wanted to ask this to all the reviewers ( @dongbohu @mhuyck @cgreene ): Right now, this bit of code (and commenting out the "$urlRouterProvider.otherwise('/home');" function on line 14 of this file) prevents the angular app from redirecting to '/#/home' whenever the url starts with '/tribe_client' (since those urls are not part of the Angular app). However, this bit of code still redirects the user to '/#/home' if no state is defined in the current url AND the url doesn't start with 'tribe_client'.

This approach seems pretty hacky to me though, but I couldn't find an 'official' way of avoiding redirection with ui-router if the url starts with 'tribe_client'. Does anybody have any more ideas on this? Thanks!